SU-HSDO/suhumsci
# SHS-92: Codeception: Spotlight Slider

# READY FOR REVIEW

## Summary
Adds a codeception functional test that verifies the spotlight slider is present and functioning.

## Urgency
medium

## Steps to Test
1. Run  codeception tests eeither via CI or locally
2. Verify that the testSpotlightSlider test passes

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
